editor/components/rich-text/index.js

```diff
@@ -426,6 +426,10 @@ export class RichText extends Component {
 	 * @return {{top: number, left: number}} The desired position of the toolbar.
 	 */
 	getFocusPosition( position ) {
+		// if the node was removed or replaced return immediately.
+		if ( ! this.containerRef.current ) {
```

Comment:
I'm surprised this even happens. Maybe worth finding out the root cause of this and comment. It might even be the same as #6981.

Reply:
It was not related to https://github.com/WordPress/gutenberg/pull/6981 I added it was a patch and it did not fix the problem here. But now that we reuse the UID this problem does not happen and all the changes to TinyMCE here reverted.